configuration/Makefile

```diff
@@ -81,5 +81,4 @@ $(CERT_DIR):
 $(GENERATE_TLS_CERT): $(BINGO_DIR)/api.mod
 	@# Install binary/ries using Go 1.14+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
 	@echo "(re)installing $(GOBIN)/generate-tls-cert"
-	@cd $(BINGO_DIR) && $(GO) build -mod=mod -modfile=api.mod -tags=tools -o=$(BIN_DIR)/generate-tls-cert "github.com/observatorium/observatorium/test/tls"
-
+	@cd $(BINGO_DIR) && $(GO) build -mod=mod -modfile=api.mod -tags=tools -o=$(BIN_DIR)/generate-tls-cert "github.com/observatorium/api/test/tls"
```

Comment:
This was the cause of the incorrect TLS certificate. The api.mod file pinned the version of observatorium/api, but the makefile wanted observatorium/observatorium (which was the old name for api). This caused bingo to resolve to tag v0.1.1, which did not support the `--server-sans` flag.